requirements.dev

```diff
@@ -0,0 +1,313 @@
+#
```

Comment:
there's only one `requirements.in` file and it looks like this uses the same `requirements.in` as `requirements.txt`. what's the difference with this one?

maybe rename the normal `requirements.txt` to `requirements.base.txt` and then create `requirements.txt` with:
```
-r requirements.base.txt
file:/shared#egg=shared
```
so that it includes the normal deps and then adds shared. and for dev builds we just install the base reqs. we'd update `requirements.base.txt` with `pip-compile --output-file requirements.base.txt requirements.in`

Reply:
The goal was to have all of the dependencies of shared be taken into account when doing the pip-compile so that there aren't any version conflicts, but that would require having shared in the requirements.in when we compile it, which complicates things slightly. I agree that we should have a requirements.base.txt but for the compilation step I'm not sure what to do.